lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,214 @@
+"""Edx branding API
+"""
+import logging
+import os
+from django.conf import settings
+from django.utils.translation import ugettext as _
+from django.shortcuts import render_to_response
+from django.core.cache import cache
+from staticfiles.storage import staticfiles_storage
+
+from microsite_configuration import microsite
+from edxmako.shortcuts import marketing_link
+
+log = logging.getLogger("edx.footer")
+
+
+def get_footer_json():
+    """ Get the footer links json
+
+    Returns:
+        Dict of footer links
+    """
+    is_edx_domain = settings.FEATURES.get('IS_EDX_DOMAIN', False)
+    key = "footer/json/{domain}".format(domain="edx" if is_edx_domain else "open_edx")
+    footer_json = cache.get(key)
+    if footer_json:
+        return footer_json
+    site_name = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+
+    context = dict()
+    context["copy_right"] = copy_right(is_edx_domain)
+    context["heading"] = heading(is_edx_domain)
+    context["logo_img"] = get_footer_logo(site_name, is_edx_domain)
+    context["social_links"] = social_links()
+    context["about_links"] = about_edx_link()
+    context["mobile_urls"] = get_mobile_urls()
+    footer_json = {"footer": context}
+    cache.set(key, footer_json)
+    return footer_json
+
+
+def copy_right(is_edx_domain):
+    """ Returns the copy rights text
+    """
+    if is_edx_domain:
+        data = _(
+            "(c) 2015 edX Inc. EdX, Open edX, and the edX and Open edX logos "
+            "are registered trademarks or trademarks of edX Inc."
+        )
+    else:
+        data = _(
+            "EdX, Open edX, and the edX and Open edX logos are registered trademarks or "
+            "trademarks of {link_start}edX Inc.{link_end}"
+        ).format(
+            link_start=u"<a href='https://www.edx.org/'>",
+            link_end=u"</a>"
+        )
+
+    return data
+
+
+def heading(is_edx_domain):
+    """ Returns the heading text copy
+    """
```

Comment:
Clean this docstring up as follows:
```python
"""Return heading copy."""
```
The words "text" and "copy" are redundant.